cache/s3proxy/s3proxy.go

```diff
@@ -196,6 +196,19 @@ func (c *s3Cache) Put(ctx context.Context, kind cache.EntryKind, hash string, si
 	}
 }
 
+func (c *s3Cache) CopyObject(ctx context.Context, bucket string, object string) {
```

Comment:
Maybe the function can be renamed to `BumpModificationTimestamp` or `UpdateModificationTimestamp` so that the name contained the information of what the method actually does.